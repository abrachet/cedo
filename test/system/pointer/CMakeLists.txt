add_cedo_system_test(basic_pointer_test.c basic_pointer_test.cedo.c a)
