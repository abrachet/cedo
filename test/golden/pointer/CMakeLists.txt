add_cedo_golden_test(CEDO_SRC input_sym.c EXPECTED_OUTPUT input_sym.s SYMS a b c)
